src/backend/marsha/core/serializers/playlist.py

```diff
@@ -26,15 +31,49 @@ class Meta:
             "users",
         ]
 
-    def create(self, validated_data):
-        """Create the Playlist object with the passed data."""
-        return Playlist.objects.create(**validated_data)
+    portable_to = serializers.SerializerMethodField(read_only=False)
+
+    def get_portable_to(self, obj):
+        """Getter for portable_to attribute instead of PlaylistSerializer to prevent recursion."""
+        return obj.portable_to.values("id", "title")
+
+    def to_internal_value(self, data):
+        """
+        Validate portable_to.
+
+        Raise a ValidationError if portable_to contains current object playlist id
+        or an unknown playlist id.
+        """
+        value: OrderedDict = super().to_internal_value(data)
+        if "portable_to" in data.keys():
+            reachable_from_playlist_ids = data.get("portable_to")
+
+            if str(self.instance.id) in reachable_from_playlist_ids:
+                raise ValidationError(_("Playlist is not portable to itself."))
+
+            try:
+                existing_playlist_ids = {
+                    str(playlist_id)
+                    for playlist_id in Playlist.objects.filter(
+                        id__in=reachable_from_playlist_ids
+                    ).values_list("id", flat=True)
+                }
+            except (Playlist.DoesNotExist, ValidationError) as error:
+                raise ValidationError(", ".join(error.messages)) from error
```

Comment:
I  don't think there will ever be these errors:
```suggestion
            existing_playlist_ids = {
                str(playlist_id)
                for playlist_id in Playlist.objects.filter(
                    id__in=reachable_from_playlist_ids
                ).values_list("id", flat=True)
            }
```

Reply:
Indeed, Playlist.DoesNotExist can't be raised here, but in case we send something that isn't a uuid in portabde_to, ValidationError is raised.
I'll remove Playlist.DoesNotExist

Reply:
Ah ok! I'm surprised the query doesn't just return an empty result, but ok then!